### DISCUSSION TITLE
Middleware per route

### DISCUSSION THREAD

**Discussion by pawmanaloto** [nitrojs/nitro#1744] [Q&A]
Does Nitro support middleware per route like express?

**Comment by robokozo** [comment]
https://nitro.unjs.io/guide/routing#route-middleware

**Comment by DavidDeSloovere** [comment]
Currently not support, you would need to check context youself. A more direct link is https://nitro.unjs.io/guide/routing#request-filtering

Please make the question as answered of this is resolved for you.
